GeometryCore/include/AABB.h

```diff
@@ -30,6 +30,9 @@ namespace geom_utils
         // Return true if p is inside the AABB boundaries.
         bool contains(const FPoint2D& p) const;
 
+        //return true if aabb2d (Bound Box) valid
+        bool isValid(const AABB2D& aabb2d);
```

Comment:
Sorry, I didn't pay attention in the previous time.. why this method take as parameter another AABB2D object? It should check `this` object - object which called method.